lib/cc/engine/analyzers/violation.rb

```diff
@@ -88,6 +95,16 @@ def read_up_path
           )
         end
 
+        def fingerprint
+          digest = Digest::MD5.new
+          digest << current_sexp.file
+          digest << "-"
```

Comment:
Gotcha, I took that part since that's what the link you gave me did as well. I'll remove em'.